### PR TITLE
foxess update signature to header for news foxcloud

### PR DESCRIPTION
**Pull request by goodsat**

### CODE REVIEW

File: custom_components/foxess/sensor.py [hunk 1]
```diff
@@ -100,6 +101,31 @@
 
 token = None
 
+async def async_get_headers(url_path, token):
+    user_agent = user_agent_rotator.get_random_user_agent()
+    lang = "en"
+    path = url_path
+    path = path.replace("https://www.foxesscloud.com","")
+    timestamp = str(int(time.time()))
```
Comment (johannessteu) [inline custom_components/foxess/sensor.py:109]: Other implementations i've seen used 
```suggestion
    timestamp = str(int(time.time() * 1000))
```

Is this not needed at all?

**Comment by johannessteu** [comment]
Related to #206

**Comment by Kepro** [comment]
it is working! can we merge this?! :)))

**Comment by FozzieUK** [comment]
This PR has been superceded and the changes are part of the new FoxESS OpenAPI branch which will be released shortly

**Pull request closed without merging**